src/EventStore.Client.PersistentSubscriptions/EventStorePersistentSubscriptionsClient.ReplayParked.cs

```diff
@@ -0,0 +1,67 @@
+using System.Threading;
+using System.Threading.Tasks;
+using EventStore.Client.PersistentSubscriptions;
+
+#nullable enable
+namespace EventStore.Client {
+	partial class EventStorePersistentSubscriptionsClient {
+		/// <summary>
+		/// Retry the parked messages of the persistent subscription
+		/// </summary>
+		/// <param name="groupName"></param>
+		/// <param name="numberOfEvents"></param>
+		/// <param name="userCredentials"></param>
+		/// <param name="cancellationToken"></param>
+		/// <returns></returns>
+		public async Task ReplayParkedForAllAsync(string groupName, long? numberOfEvents = null,
+			UserCredentials? userCredentials = null, CancellationToken cancellationToken = default) {
+
+			var req = new ReplayParkedReq() {
+				Options = new ReplayParkedReq.Types.Options{
+					GroupName = groupName,
+					All = new Empty()
+				},
+			};
+
+			await ReplayParkedInternalAsync(req, numberOfEvents, userCredentials, cancellationToken)
+				.ConfigureAwait(false);
+		}
+
+		/// <summary>
+		/// Retry the parked messages of the persistent subscription
+		/// </summary>
+		/// <param name="groupName"></param>
+		/// <param name="stream"></param>
+		/// <param name="numberOfEvents"></param>
+		/// <param name="userCredentials"></param>
+		/// <param name="cancellationToken"></param>
+		/// <returns></returns>
+		public async Task ReplayParkedForStreamAsync(string stream, string groupName, long? numberOfEvents = null,
+			UserCredentials? userCredentials = null, CancellationToken cancellationToken = default) {
+
+			var req = new ReplayParkedReq() {
+				Options = new ReplayParkedReq.Types.Options {
+					GroupName = groupName,
+					StreamIdentifier = stream
+				},
+			};
+			
+			await ReplayParkedInternalAsync(req, numberOfEvents, userCredentials, cancellationToken)
+				.ConfigureAwait(false);
+		}
+		
+		private async Task ReplayParkedInternalAsync(ReplayParkedReq req, long? numberOfEvents ,
+			UserCredentials? userCredentials, CancellationToken cancellationToken) {
+
+			if (numberOfEvents.HasValue) {
+				req.Options.StopAt = numberOfEvents.Value;
+			} else {
+				req.Options.NoLimit = new Empty();	
+			}
+			
+			await new PersistentSubscriptions.PersistentSubscriptions.PersistentSubscriptionsClient(
```

Comment:
**QUESTION:** Shouldn't we have here the HTTP fallback?

Reply:
Done.